sdk/framework/backend.go

```diff
@@ -528,9 +528,20 @@ func (b *Backend) handleRootHelp(req *logical.Request) (*logical.Response, error
 	// names in the OAS document.
 	requestResponsePrefix := req.GetString("requestResponsePrefix")
 
+	// Generic paths will primarily be used for code generation purposes.
+	// This will result in a dynamic mount path being placed instead of
+	// a hardcoded default path. For example /auth/approle/login would be replaced
```

Comment:
```suggestion
	// Generic mount paths will primarily be used for code generation purposes.
	// This will result in a dynamic mount paths being placed instead of
	// a hardcoded default paths. For example /auth/approle/login would be replaced
```